src/index.tsx

```diff
@@ -1,16 +1,16 @@
+import dotenv from 'dotenv';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import dotenv from 'dotenv';
+import { ApolloProvider } from '@apollo/react-hooks';
+import client from './services/ApolloClient';
 import './index.css';
 import App from './App';
-import * as serviceWorker from './serviceWorker';
 
 dotenv.config();
```

Comment:
You don't need to use dotenv since CRA read for you the env variables, i see you removed the package, remember to get rid of this line.